tests/utests/basic/test_inout.c

```diff
@@ -44,6 +44,18 @@ test_input_mem(void **UNUSED(state))
     ly_in_free(in, 0);
 }
 
+#ifdef _WIN32
+int invalid_parameter_handler_called = 0;
+void empty_invalid_parameter_handler(const wchar_t* expression,
+   const wchar_t* function,
+   const wchar_t* file,
+   unsigned int line,
+   uintptr_t pReserved)
+{
+    ++invalid_parameter_handler_called;
+}
+#endif
+
```

Comment:
For all I care you can remove this test instead of this wild code.

Reply:
It was there to check that the FD is not leaked, but I agree, that's a super-minor thing. Gone.